src/Essentials/src/Geolocation/Geolocation.shared.cs

```diff
@@ -35,6 +35,11 @@ public interface IGeolocation
 		/// </summary>
 		bool IsListeningForeground { get; }
 
+		/// <summary>
+		/// Returns true when the device's GPS is enabled
```

Comment:
Lets not use GPS specifically, other methods are used to determine the location. Maybe "location services"?